test/e2e/View.test.ts

```diff
@@ -0,0 +1,131 @@
+import Graph from '../../src/component/graph/Graph';
+
+describe('BPMN Visu JS', () => {
+  // region html string literal
+  const xmlContent = `
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
:question: are we sure we want to use data from bpmn-js :smiling_imp: ?

Reply:
changed to the consortium one